app/controllers/concerns/prominence_headers.rb

```diff
@@ -0,0 +1,34 @@
+##
+# Set any headers related to prominence.
+#
+module ProminenceHeaders
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :set_prominence_headers
+  end
+
+  def has_prominence
```

Comment:
Not sure about this method name either. It's basically saying `def the_thing_that_has_prominence_that_sets_visibility_for_the_record_we_are_accessing` but obviously not great.

Prominence is _almost_ a polymorphic association, except with no backend. `prominenceable`? IDK, open to suggestion here.

Reply:
Yes, feels like we're close to making a Prominence model. Maybe not for now but to consider for the future